<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QWidget" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>88</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="sliderRed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Red:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>66</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Green:</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderGreen">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue:</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderBlue">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorWidget</receiver>
   <slot>changedRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorWidget</receiver>
   <slot>changedGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorWidget</receiver>
   <slot>changedBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
